### ISSUE TITLE
Multiple sessions per 1 user

### ISSUE THREAD

**Issue by xikavi** [stascorp/rdpwrap#1210]
Hi. I have Win 10 Pro 20H2 and termsrv.dll 10.0.19041.84. In RDPConf all is green, "Single session per user" flag disabled both in RDPConf and group policy. But when i trying to open multiple connections to one user, previous connection closes and says that another user connected. Is there any trick to fix it?

**Comment by tomkanok** [comment]
Are you sure that your wrapper is in action and not native Win10 Pro RDP ? Disable Win 10 Pro remote desktop integrated functionality first and try again.

**Comment by robbak** [comment]
Yes, that is normal. When you sign in using rdp, it will take over the
local session. That is by design, and a feature of rdp and not altered by
rdpwrapper.

But, with multiple sessions enabled, you should be able to create a second
rdp session with that user account.

**Comment by xikavi** [comment]
> Are you sure that your wrapper is in action and not native Win10 Pro RDP ? Disable Win 10 Pro remote desktop integrated functionality first and try again.

What do you mean? When i disable RDP in windows parameters, "listener state" changes to "not listening" and RDP stops working entirely.

> Yes, that is normal. When you sign in using rdp, it will take over the local session. That is by design, and a feature of rdp and not altered by rdpwrapper. But, with multiple sessions enabled, you should be able to create a second rdp session with that user account.

I thought so, but no, cant connect multiple times. On win 8.1 and earlier it worked btw. I'll try to use server 2019 on VM, maybe there it will work.

**Comment by tomkanok** [comment]
> What do you mean? When i disable RDP in windows parameters, "listener state" changes to "not listening" and RDP stops working entirely.

I have disabled integrated RDP hpst on Windows Pro (Settings: Enable Remote Desktop = Disabled), then installed RDP Wrapper. and enabled **more sessions per user** in RDPConf.  Now it works for me.

**Comment by xikavi** [comment]
> I have disabled integrated RDP hpst on Windows Pro (Settings: Enable Remote Desktop = Disabled), then installed RDP Wrapper. and enabled **more sessions per user** in RDPConf. Now it works for me.

Ok, thanks, it started to work randomly. Also, found interesting thing: if connecting using network name like MY-PC it allows only one session, but when connecting by ip address, then could open many sessions.
